src/hapi/pipelines/database/humanitarian_needs.py

```diff
@@ -58,26 +59,24 @@ def get_admin2_ref(self, row, dataset_name, errors):
             )
         return ref
 
-    def populate(self):
+    def populate(self) -> None:
         logger.info("Populating humanitarian needs table")
-        reader = Read.get_reader("hdx")
-        datasets = reader.search_datasets(
-            filename="Global HPC HNO*",
-            fq="name:global-hpc-hno-*",
-            configuration=self._configuration,
-        )
         warnings = set()
         errors = set()
-        for dataset in datasets:
+        reader = Read.get_reader("hdx")
+        dataset = reader.read_dataset("global-hpc-hno", self._configuration)
+        self._metadata.add_dataset(dataset)
+        dataset_id = dataset["id"]
+        dataset_name = dataset["name"]
+        for resource in dataset.get_resources():
```

Comment:
Maybe we only need to include the latest resource in HAPI? As we're doing for 3W

Reply:
Yes following from our discussion I realise this